hudi-spark/src/main/scala/org/apache/hudi/DefaultSource.scala

```diff
@@ -50,7 +50,7 @@ class DefaultSource extends RelationProvider
                               optParams: Map[String, String],
                               schema: StructType): BaseRelation = {
     // Add default options for unspecified read options keys.
-    val parameters = Map(VIEW_TYPE_OPT_KEY -> DEFAULT_VIEW_TYPE_OPT_VAL) ++: optParams
+    val parameters = Map(VIEW_TYPE_OPT_KEY -> DEFAULT_VIEW_TYPE_OPT_VAL) ++ optParams
```

Comment:
`++:` works fine in my IDEA, did you install scala lib?

Reply:
hi, @yanghua,  Can you help to verify whether this works fine in your env?